### PR TITLE
Fix settings not being loaded at CLI

### LINKED ISSUE #25898: Create org permissions of user created via cli client
### Description

With (default) settings as

```
[admin]
DISABLE_REGULAR_ORG_CREATION = false

[service]
DEFAULT_ALLOW_CREATE_ORGANIZATION = true
```

a user created via `gitea admin user create` does not have "create org" permissions. If it is intentional that the CLI client does not respect the relevant settings, it might be helpful to add an option to the CLI client.

### Gitea Version

1.19.0

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

https://gist.github.com/nbenn/f4cf2314da1cc32d8d5325b9a0e9cbe3

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

Docker: gitea/gitea:1.19.0

### Database

SQLite

### PR DESCRIPTION
**Pull request by cassiozareck**
Closes #25898
The problem was that the default settings weren't being loaded, so 
https://github.com/go-gitea/gitea/blob/ad69f7175af3dc6974df6ad22f22833c284456df/models/user/user.go#L593
was always false.

### CODE REVIEW

**Comment by silverwind** [comment]
I wonder if this is really the right place to load the setting.

**Comment by cassiozareck** [comment]
Both way works as of my tests. I thought to initialize together with the DB but thinking better at the beginning may be more clean. I'll adjust it

**Comment by silverwind** [comment]
Ah I see the context now, previous place was fine then I guess, the validation does not depend on settings it looks like. Note I'm not really knowledgeable around these parts so take my comments with a grain of salt :)

**Comment by cassiozareck** [comment]
With this change there's happening some output when a user runs `gitea admin user create` for example. Like this:
```
2023/08/11 10:34:13 ...les/setting/cache.go:75:loadCacheFrom() [I] Cache Service Enabled
2023/08/11 10:34:13 ...les/setting/cache.go:90:loadCacheFrom() [I] Last Commit Cache Service Enabled
2023/08/11 10:34:13 ...s/setting/session.go:74:loadSessionFrom() [I] Session Service Enabled
New user 'user1' has been successfully created!
```

**Comment by silverwind** [comment]
That reinforces that it needs to be after validation.

**Comment by cassiozareck** [comment]
Hmm so its ok? I think its unavoidable unless we try to change the config for logging outputs before

**Comment by silverwind** [comment]
Just revert to previous location imho.

**Comment by cassiozareck** [comment]
Oh no no! It happen doesn't matter where loadsetting is. There's log statements inside loadsettings. I think its alright as it states settings are being loaded but if you dont want log to happen I can see what I can do

File: cmd/admin_user_create.go [hunk 1]
```diff
@@ -67,6 +67,8 @@ var microcmdUserCreate = &cli.Command{
 }
 
 func runCreateUser(c *cli.Context) error {
+	setting.LoadSettings()
```
Comment (lunny) [inline cmd/admin_user_create.go:74]: It's better to move this line to line 86 and have some tests in `admin_user_create_test.go`.

Comment (lunny) [reply cmd/admin_user_create.go:74]: ~This is not right, because then some settings will be loaded twice.~

**Comment by wxiaoguang** [comment]
This fix works but it is somewhat hacky and fragile.

Ideally, there should be command line options to set the flags, but not continue loading more unclear settings.

**Review by lunny — APPROVED** [review]
As a quick fix, I think it's enough.

**Comment by wxiaoguang** [comment]
Made some changes in 170261a6a3ef2719e27ca6b8c7297cf5db85cf0b :

* do not show unnecessary logs
* add comments

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**